specification/cognitiveservices/Language.AnalyzeText-authoring/models.tsp

```diff
@@ -534,6 +534,52 @@ model TextAnalysisAuthoringTrainingJobOptions {
    * as 20.
    */
   evaluationOptions?: TextAnalysisAuthoringEvaluationOptions;
+
+  /**
+   * Represents the settings for using data generation as part of training a custom model.
+   */
+  @added(Versions.v2024_11_15_preview)
+  dataGenerationSettings?: TextAnalysisAuthoringDataGenerationSettings;
+}
+
+/**
+ * Represents the settings for using data generation as part of training a custom model.
+ */
+@added(Versions.v2024_11_15_preview)
+@clientName("DataGenerationSettings")
+model TextAnalysisAuthoringDataGenerationSettings {
+  /**
+   * If set to true, augment customer provided training data with synthetic data to improve model quality.
+   */
+  enableDataGeneration: boolean;
+
+  /**
+   * Represents the connection info for the Azure resource to use during data generation as part of training a custom model.
+   */
+  @added(Versions.v2024_11_15_preview)
+  dataGenerationConnectionInfo: TextAnalysisAuthoringDataGenerationConnectionInfo;
+}
+
+/**
+ * Represents the connection info for the Azure resource to use during data generation as part of training a custom model.
+ */
+@added(Versions.v2024_11_15_preview)
+@clientName("DataGenerationConnectionInfo")
+model TextAnalysisAuthoringDataGenerationConnectionInfo {
```

Comment:
Hi Mark, thanks for your comment. The reason we added the TextAnalysisAuthoring prefix is to distinguish these models from those in the ConversationalAnalysisAuthoring service. Since many models across both services share similar names, using distinct prefixes helps tell them apart.

Reply:
But these are in separate SDKs, and you are calling them the same thing in the client.  If you used them in the same TypeSpec project, this would make sense, but simplifying the client name indicates you can simplify the name altogether.